less/components/header.less

```diff
@@ -36,31 +36,32 @@
       display: inline-block;
       color: inherit;
       font-weight: 700;
+      height: 18px;
```

Comment:
Let's stay away from fixed pixel values; probably best to use `em`s here since it's tied to the font height. Same for below